src/test/java/com/puppycrawl/tools/checkstyle/utils/XpathUtilTest.java

```diff
@@ -72,6 +79,68 @@ public void testGetValue() {
                 getTextAttributeValue(createDetailAST(TokenTypes.STRING_LITERAL, "HELLO WORLD")));
     }
 
+    @Test
+    public void testPrintXpathOne() throws Exception {
+        final String eol = System.lineSeparator();
+        final String fileContent = "class Test { public void method() {int a = 5;}}";
+        final Path path = Files.createTempFile("", "");
+        Files.write(path, fileContent.getBytes(StandardCharsets.UTF_8));
+        final String expected = "CLASS_DEF -> CLASS_DEF [1:0]" + eol
+            + "`--OBJBLOCK -> OBJBLOCK [1:11]" + eol
+            + "    |--METHOD_DEF -> METHOD_DEF [1:13]" + eol
+            + "    |   `--SLIST -> { [1:34]" + eol
+            + "    |       |--VARIABLE_DEF -> VARIABLE_DEF [1:35]" + eol
+            + "    |       |   |--IDENT -> a [1:39]" + eol;
+        final String result = XpathUtil.printXpathBranch(
+            "//CLASS_DEF//METHOD_DEF//VARIABLE_DEF//IDENT", path.toFile());
+        Files.delete(path);
```

Comment:
Yeah, I forgot to change it.

By the way, test you are referring to fails on my machine (win 10) and it is really annoying and slows down development, it should be changed somehow. I will created an issue for it.

Reply:
please always report issues in build. Full build should pass on windows without any problems.
I have linux and what is see on CI. 
Build should pass on all OSs and all jdk, we try to keep it actual and always green.

Please find some time to report details, and probably share what is not working, ideally find a way to change something in Appveyor CI to make it reproducible in CI, so it will not be a problem for you anymore.

Reply:
I added tempFolder rule for temp file creation.

About issues - I created #7032 for it